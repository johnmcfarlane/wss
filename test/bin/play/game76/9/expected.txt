 69  9,1  | JARGLE
 50  9,5  | RAJ
 48  6,2  - AGLEAF
 48  9,7  | JET
 45  9,1  | ALARGE
 42  9,1  | AGLARE
 42  9,1  | GALERA
 41  1,7  | LEG
 39  9,3  | JAGER
 39  8,4  | JAGERS
 39  8,4  | JAGRAS
 38  1,7  | GEL
 38  1,7  | REG
 37  4,10 - AJAR
 37  9,9  | TAJ
 37  9,9  | TEJ
 35  8,4  - JGER
 34  9,3  | JAGA
 33  9,4  | AJAR
 33  9,7  | RETAG
 32  9,5  | JAR
 32 10,1  | JE
 32  1,7  | REL
 30  9,3  | JARA
 30  8,7  - JGER
 30  9,3  | RAJA
 30  9,7  | RATAL
 30  9,7  | RATEL
 30  9,7  | RETAL
 29  1,8  | EL
 29  1,7  | LEA
 29  1,7  | LER
 28  9,4  | JGER
 27  9,5  | JA
 27  7,3  - JALEO
 27  5,10 - JAR
 27  9,5  | JE
 27  6,7  - KEGLER
 26  1,7  | AER
 26  9,2  | JAGLA
 26  8,7  - JEER
 26 10,1  | LEG
 26  1,7  | REA
 25 10,1  | ARLE
 25 10,1  | GEL
 24  9,2  | ALEGAR
 24 10,1  | ARG
 24 10,1  | ERG
 24 10,3  | JA
 24  9,2  | JAGRA
 24 10,3  | JE
 24 10,1  | REG
 23  4,10 - AGAR
 23 10,2  | ELA
 23  1,8  | ER
 23  9,3  | GALEA
 23  9,2  | LAAGER
 23  9,3  | LAGER
 23 10,2  | RLE
 22  9,3  | ALGAE
 22 10,1  | AREA
 22 10,1  | GRE
 22  5,10 - JA
 22  8,3  | JAEGARS
 22  5,10 | JO
 21  8,7  - AGER
 21  4,10 - ALAR
 21  8,7  - EGER
 21  8,6  | JAGS
 21  9,7  - JE
 21 10,1  | REL
 20  2,8  - BENJY
 20 10,2  | REA
 19  9,4  | AGAR
 19  9,4  | AGER
 19  9,6  | ARET
 19 10,2  | EL
 19  9,6  | ERAT
 19  8,7  - GEER
 19  9,4  | GLAR
 19  8,4  | GRAALS
 19  8,5  | JARLS
 19  1,7  | JE
 18  4,10 - AGA
 18 10,1  | ARE
 18  9,8  | ATAR
 18  9,5  | GAE
 18  9,3  | GALA
 18  9,3  | GALE
 18  9,5  | GAR
 18  8,2  - GRAF
 18  7,4  - JAGER
 18  9,2  | LARGE
 18  8,7  - LEER
 18  9,7  - LER
 17  9,4  | ALAE
 17  9,4  | ALAR
 17  9,4  | ALEA
 17  9,3  | ALGA
 17  9,2  | ALGAE
 17  9,2  | ARAGE
 17  9,2  | ARGLE
 17  9,4  | GAEA
 17  8,4  | GALEAS
 17  9,2  | GARLE
 17  9,4  | GEAR
 17  9,2  | GLARE
 17  8,4  | GLARES
 16  9,7  - AER
 16  9,3  | AGLA
 16  4,10 - ALA
 16  8,2  - ALEF
 16  8,3  | ALEGARS
 16  9,7  - EER
 16  4,10 - ELA
 16  9,3  | GAEA
 16  9,2  | GALEA
 16  9,3  | GARA
 16  9,3  | GARE
 16  8,6  - JAPE
 16  9,5  | LAR
 16  9,5  | LEA
 16  8,2  - LEAF
 16  9,4  | LEAR
 16  9,5  | LER
 16  9,3  | RAGA
 16  9,3  | RAGE
 16  8,5  | RAJAS
 16  8,2  - RALF
 16 10,2  | RE
 16  9,9  | TAG
 16  9,9  | TEG
 15  8,4  - AJAR
 15  9,4  | AREA
 15  8,4  | ARGALS
 15  1,9  - GIT
 15  8,6  | JARS
 15  8,4  | LAGERS
 15  8,4  | LARGES
 14 10,1  | AE
 14  9,5  | AER
 14  9,4  | AGA
 14  9,4  | AGE
 14 10,1  | AR
 14  9,3  | ARLE
 14  9,5  | EAR
 14  9,2  - ELF
 14 10,1  | ER
 14 10,7  - ER
 14 11,2  | FORCER
 14  8,3  | LAAGERS
 14  9,3  | LARE
 14  9,3  | RALE
 14  9,7  | RATA
 14  9,7  | RATE
 14 10,1  | RE
 14  9,5  | REA
 13  9,3  | ALAE
 13  9,3  | ALEA
 13  8,4  | ARGLES
 13  5,10 - GAR
 13  8,4  - GLAR
 13  8,7  - JEE
 13  8,6  | LAGS
 13  8,6  | LEGS
 13  9,9  | TAL
 13  9,9  | TEL
 12  9,6  | AA
 12  9,6  | AE
 12  9,7  | AET
 12  9,4  | ALA
 12  9,4  | ALE
 12  9,6  | AR
 12  9,3  | AREA
 12  9,2  - ARF
 12  9,7  | EAT
 12  9,4  | ELA
 12  9,6  | ER
 12  9,2  - ERF
 12 11,9  | ERG
 12  8,6  | ERGS
 12  9,4  | GAE
 12  8,5  | GAELS
 12  8,4  - GEAR
 12  9,4  | GRA
 12  9,4  | GRE
 12  9,4  - JAR
 12  1,9  - LIT
 12  8,6  | RAGS
 12  9,7  | RAT
 12  9,2  - REF
 12  9,7  | RET
 12  9,4  | RLE
 11  8,4  | ALARES
 11  6,4  - ALEGAR
 11  8,4  | ARALES
 11 11,8  | EEL
 11  8,6  | GALS
 11  8,6  | GELS
 11 10,3  - JO
 11  9,5  | LA
 11  6,4  - LAAGER
 11  5,10 - LAR
 11  6,4  - LARGER
 11  7,3  - LARGO
 11  9,4  | LEA
 11  6,3  - REGALO
 10  1,7  | AE
 10  8,5  | ALGAS
 10  9,4  | ARA
 10  9,4  | ARE
 10  9,8  - AWA
 10  9,8  - AWE
 10  8,5  | EARLS
 10 10,2  - EF
 10  9,4  | ERA
 10  8,5  | GALAS
 10  8,5  | GALES
 10  7,4  - LAGER
 10  1,7  | RE
 10  9,4  | REA
 10  8,5  | REALS
 10 10,2  - RF
 10  9,9  | TAA
 10  9,9  | TAE
 10  9,9  | TAR
 10  9,9  | TEA
 10  9,9  | TRA
  9  9,5  | AA
  9  8,6  | AALS
  9  9,5  | AE
  9  8,3  - AGAO
  9  8,5  | AGARS
  9  8,5  | AGERS
  9  1,9  - AIT
  9  8,5  | ARGAS
  9  9,8  | ATE
  9  4,10 | AX
  9  5,10 - EAR
  9 11,9  | EEL
  9 11,8  | EER
  9  8,7  | ELSA
  9  9,8  | ETA
  9  4,10 | EX
  9  8,6  - GAPE
  9  8,6  | GARSE
  9  8,5  | GEARS
  9  6,7  - KEG
  9  6,7  - KGR
  9  8,4  - LEAR
  9  8,5  | RAGAS
  9  7,4  - RAGER
  9  8,5  | RAGES
  9  8,6  | RALS
  9  9,5  | RE
  9  1,9  - RIT
  8  8,4  - AGAR
  8  8,6  | AGAS
  8  8,7  - AGE
  8  8,4  - AGER
  8  8,6  | AGES
  8  8,7  | ALS
  8  8,5  | ARLES
  8  1,8  - EBEN
  8 11,9  | ELA
  8  8,7  | ELS
  8  9,10 - EeL
  8  8,6  | GAES
  8  8,6  | GARS
  8  9,3  - GEO
  8  8,6  | GERS
  8  8,6  | GRAS
  8  8,6  | GRES
  8  9,3  - GRO
  8  6,7  - KAL
  8 10,3  | LA
  8  8,5  | LARES
  8  8,5  | LEARS
  8  8,5  | RALES
  8  8,4  - REGR
  7  8,4  - ALAR
  7  8,6  | ALAS
  7  8,6  | ALES
  7  8,6  | ARASE
  7  7,4  - AREAR
  7  8,5  | AREAS
  7  8,3  - ARGO
  7  8,8  | ASG
  7  9,8  | AT
  7  9,8  - AW
  7 11,8  | EE
  7  8,3  - ERGO
  7  9,8  | ET
  7  9,8  - EW
  7  8,3  - GARO
  7  8,7  | GAS
  7  8,7  | GES
  7  8,7  | GRS
  7  6,7  - KAE
  7  6,7  - KEA
  7  6,7  - KER
  7  6,7  - KRA
  7  8,6  | LARS
  7  8,7  | LASA
  7  8,7  | LASE
  7  8,6  | LEAS
  7  8,6  - RAPE
  7  8,3  - REGO
  7 10,8  - WE
  6  9,3  - AGO
  6  8,7  - ALE
  6  9,6  - APE
  6  6,10 - AR
  6  8,4  - ARAR
  6  8,6  | ARES
  6  8,7  | ARSE
  6  9,10 - AeR
  6  8,6  | EARS
  6 11,9  | EER
  6  9,3  - EGO
  6  9,6  - EPE
  6 11,9  | ERA
  6  8,6  | ERAS
  6 11,9  | ERE
  6  9,10 - EeR
  6  8,7  - GAE
  6  8,7  - GEE
  6  8,7  - GRE
  6  6,7  - KA
  6  5,10 - LA
  6  9,3  - LAO
  6  8,7  | LAS
  6  9,3  - LEO
  6  5,10 | LO
  6  8,7  | RASA
  6  8,7  | RASE
  6  8,4  - REAR
  6  8,7  - RLE
  6  7,9  | URS
  5  8,7  | AAS
  5  8,3  - AERO
  5  8,7  | AES
  5  9,4  - AGR
  5  8,7  | ARS
  5  8,8  | ASA
  5  8,8  | ASE
  5 11,9  | EL
  5  8,7  | ERS
  5  9,4  - GAR
  5  9,4  - GRR
  5  8,7  - LEE
  5  8,7  | RAS
  5  8,7  | RES
  5 10,10 - eL
  4  8,7  - ARE
  4  9,3  - ARO
  4  8,8  | AS
  4  8,7  - ERE
  4  8,8  | ES
  4 10,3  - GO
  4  9,4  - LAR
  4  9,4  - LER
  4  8,7  - REE
  3  9,7  - AE
  3  9,4  - AER
  3  9,4  - ARR
  3  9,10 - Ae
  3  9,4  - EAR
  3  9,7  - EE
  3 11,9  | EE
  3 11,9  | ER
  3  9,4  - ERR
  3  9,10 - Ee
  3 10,3  - LO
  3  9,7  - RE
  3  9,9  | TA
  3  9,9  | TE
  3 10,10 - eE
  3 10,10 - eR
  2 10,4  - AR
  2 10,3  - EO
  2 10,4  - ER
