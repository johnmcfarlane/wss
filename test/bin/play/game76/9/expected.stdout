 50  9,5  | RAJ
 39  8,4  | JAGERS
 39  8,4  | JAGRAS
 37  9,9  | TAJ
 33  9,4  | AJAR
 32  9,5  | JAR
 32 10,1  | JE
 30  9,3  | RAJA
 30  9,7  | RATAL
 30  9,7  | RATEL
 27  9,5  | JA
 27  6,7  - KEGLER
 26  8,7  - JEER
 26 10,1  | LEG
 25 10,1  | GEL
 24  9,2  | ALEGAR
 24 10,3  | JA
 24  9,2  | JAGRA
 24 10,3  | JE
 24 10,1  | REG
 22  9,3  | ALGAE
 22  5,10 - JA
 22  5,10 | JO
 21  8,7  - AGER
 21  8,7  - EGER
 21  8,6  | JAGS
 21  9,7  - JE
 19  9,4  | AGAR
 19 10,2  | EL
 19  8,4  | GRAALS
 19  8,5  | JARLS
 18  4,10 - AGA
 18  9,5  | GAE
 18  9,3  | GALA
 18  9,5  | GAR
 18  7,4  - JAGER
 18  8,7  - LEER
 17  9,4  | ALAE
 17  9,4  | ALAR
 17  9,3  | ALGA
 17  8,4  | GALEAS
 17  9,4  | GEAR
 17  8,4  | GLARES
 16  4,10 - ALA
 16  8,2  - ALEF
 16  8,3  | ALEGARS
 16  9,2  | GALEA
 16  8,6  - JAPE
 16  9,5  | LAR
 16  8,2  - LEAF
 16  9,4  | LEAR
 16  9,3  | RAGA
 16  8,5  | RAJAS
 16  8,5  | RAJES
 16  9,9  | TAG
 16  9,9  | TEG
 15  8,4  - AJAR
 15  8,4  | ARGALS
 15  1,9  - GIT
 15  8,6  | JARS
 15  8,4  | LAGERS
 15  8,4  | LARGES
 14 10,1  | AE
 14  9,4  | AGA
 14  9,5  | EAR
 14  9,2  - ELF
 14 10,7  - ER
 14 11,2  | FORCER
 14  8,3  | LAAGERS
 14  9,7  | RATE
 14 10,1  | RE
 13  8,4  | ARGLES
 13  8,7  - JEE
 13  8,6  | LAGS
 13  8,6  | LEGS
 13  9,9  | TEL
 12  9,6  | AA
 12  9,6  | AE
 12  9,4  | ALA
 12  9,6  | AR
 12  9,3  | AREA
 12  9,2  - ARF
 12  9,7  | EAT
 12 11,9  | ERG
 12  8,6  | ERGS
 12  8,4  - GEAR
 12  9,4  - JAR
 12  1,9  - LIT
 12  8,6  | RAGS
 12  9,7  | RAT
 12  9,2  - REF
 12  8,6  | REGS
 11  8,7  | AGS
 11  6,4  - ALEGAR
 11 11,8  | EEL
 11  8,6  | GALS
 11  8,6  | GELS
 11 10,3  - JO
 11  9,5  | LA
 11  6,4  - LAAGER
 11  6,4  - LARGER
 11  7,3  - LARGO
 11  9,4  | LEA
 10  8,5  | ALGAS
 10  9,8  - AWA
 10  9,8  - AWE
 10  8,5  | EARLS
 10 10,2  - EF
 10  9,4  | ERA
 10  8,5  | GALAS
 10  8,5  | GALES
 10  7,4  - LAGER
 10  8,5  | REALS
 10  9,9  | TAE
 10  9,9  | TAR
 10  9,9  | TEA
  9  9,5  | AA
  9  8,6  | AALS
  9  8,5  | AGARS
  9  8,5  | AGERS
  9  1,9  - AIT
  9  9,8  | ATE
  9  4,10 | AX
  9 11,9  | EEL
  9  4,10 | EX
  9  8,6  - GAPE
  9  8,5  | GEARS
  9  6,7  - KEG
  9  8,4  - LEAR
  9  8,5  | RAGAS
  9  8,5  | RAGES
  8  8,4  - AGAR
  8  8,6  | AGAS
  8  8,7  - AGE
  8  8,4  - AGER
  8  8,6  | AGES
  8  8,7  | ALS
  8  8,5  | ARLES
  8  8,7  | ELS
  8  9,10 - EeL
  8  8,6  | GAES
  8  8,6  | GARS
  8 10,3  | LA
  8  8,5  | LARES
  8  8,5  | LEARS
  8  8,5  | RALES
  7  8,4  - ALAR
  7  8,6  | ALAS
  7  8,6  | ALES
  7  8,5  | AREAS
  7  9,8  | AT
  7  9,8  - AW
  7 11,8  | EE
  7  8,3  - ERGO
  7  8,7  | GAS
  7  6,7  - KAE
  7  6,7  - KEA
  7  8,6  | LARS
  7  8,7  | LASE
  7  8,6  | LEAS
  7  8,6  - RAPE
  7 10,8  - WE
  6  9,3  - AGO
  6  8,7  - ALE
  6  9,6  - APE
  6  8,6  | ARES
  6  8,7  | ARSE
  6  8,6  | EARS
  6  9,3  - EGO
  6 11,9  | ERA
  6  8,6  | ERAS
  6 11,9  | ERE
  6  8,7  - GAE
  6  8,7  - GEE
  6  6,7  - KA
  6  5,10 - LA
  6  8,7  | LAS
  6  5,10 | LO
  6  8,7  | RASE
  6  8,4  - REAR
  5  8,7  | AAS
  5  8,3  - AERO
  5  8,7  | ARS
  5 11,9  | EL
  5  8,7  | ERS
  5  9,4  - GAR
  5  8,7  - LEE
  5  8,7  | RAS
  5  8,7  | RES
  5 10,10 - eL
  4  8,7  - ARE
  4  8,8  | AS
  4  8,7  - ERE
  4  8,8  | ES
  4 10,3  - GO
  4  9,4  - LAR
  4  8,7  - REE
  3  9,7  - AE
  3  9,10 - Ae
  3  9,4  - EAR
  3  9,7  - EE
  3 11,9  | EE
  3 11,9  | ER
  3  9,4  - ERR
  3  9,10 - Ee
  3 10,3  - LO
  3  9,7  - RE
  3  9,9  | TA
  3  9,9  | TE
  3 10,10 - eE
  3 10,10 - eR
  2 10,4  - AR
  2 10,4  - ER
